examples/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+# setup tiny-cnn
+include_directories(${PROJECT_SOURCE_DIR}) 
+link_libraries(${REQUIRED_LIBRARIES})
+
+add_executable(benchmarks benchmarks/main.cpp ${tiny_cnn_hdrs})
+add_executable(example_mnist_train mnist/train.cpp ${tiny_cnn_hrds})
+target_link_libraries(example_mnist_train ${REQUIRED_LIBRARIES})
+
+add_executable(example_deconv_train deconv/train.cpp ${tiny_cnn_hrds})
+target_link_libraries(example_deconv_train ${REQUIRED_LIBRARIES})
+
+if(USE_OPENCV)
+    add_executable(example_mnist_test mnist/test.cpp ${tiny_cnn_hdrs})
+    add_executable(example_deconv_visual deconv/visual.cpp ${tiny_cnn_hdrs})
+    target_link_libraries(example_mnist_test ${REQUIRED_LIBRARIES})
+    target_link_libraries(example_deconv_visual ${REQUIRED_LIBRARIES})
+else()
+    #file(GLOB stb_image_hdrs mnist/stb_image/*.h)
```

Comment:
@edgarriba 
I think stb version shouldn't be removed, and I've fixed the compiler error in stb_image version in #219. You can uncomment these lines later.